cluster/gce/manifests/glbc.manifest

```diff
@@ -15,7 +15,7 @@ spec:
   terminationGracePeriodSeconds: 600
   hostNetwork: true
   containers:
-  - image: gcr.io/k8s-ingress-image-push/ingress-gce-glbc-amd64:v1.12.0
+  - image: gcr.io/k8s-ingress-image-push/ingress-gce-glbc-amd64:nightly
```

Comment:
fixing the test lgtm, but pointing this at a floating tag does not seem correct

Reply:
done